hibike/lib/hibike/hibike_device.cpp

```diff
@@ -141,3 +113,12 @@ void toggleLED() {
     led_enabled = true;
   }
 }
+
+uint64_t getLastHeartbeatResponse(){
```

Comment:
Are these functions used?

From a coder's perspective, it'd be more useful to know how long it's been since we got a heartbeat, not the absolute time that we last recieved our heartbeat at.